[IAM]Update scope.md (#12781)
cloudflare/cloudflare-docs: content/fundamentals/setup/manage-members/scope.md

You choose the scope of a role when you [add a member](/fundamentals/setup/manage-members/manage/) to your account.

> {{<Aside type="note">}}
> 
> You need to choose to assign either Account Scope roles or Domain Scope role. You will not be able to combine both scope roles for the same member. 
> 
> {{</Aside>}}
> 

Reviewer: ```suggestion
{{<Aside type="note">}}
You need to choose to assign either an Account Scope role or Domain Scope role. You cannot combine both scope roles for the same member. 
{{</Aside>}}

```
